src/components/pages/CaseDetails/RenderCaseDetails.js

```diff
@@ -11,22 +12,22 @@ const RenderCaseDetails = () => {
 
   return (
     <div className="RenderLayoutTemplate">
-      <Layout className="Layout">
-        <Header className="Header">
-          <NavHeader />
-        </Header>
-        <Content className="Content">
-          <BreadCrumbs pathname={location.pathname} />
-          <div className="Content__Container">
-            <Card className="Content__Container-Card">
-              <CaseDetails />
-            </Card>
-          </div>
-        </Content>
-        <Footer className="Footer">
-          Family Promise of Spokane ©2022 Created by BloomTech Labs
-        </Footer>
-      </Layout>
+      {/* <Layout className="Layout"> */}
+      {/* <Header className="Header"> */}
```

Comment:
How come you commented this out and not just deleted it?

Reply:
I did delete it, not sure why it's showing as commented here, I might've forgot to add that before I pushed the draft PR, will fix it now!